cmd/flow-dps-indexer/README.md

```diff
@@ -0,0 +1,26 @@
+# Flow DPS Indexer
```

Comment:
I agree partially. `flow-rosetta-server` makes sense. The rest, imo, does not.

Regarding removing the `flow-` prefix, you should think about these folder names as the default name that will be used for the binaries across all users. Within the repository, it makes sense. However, having `dps-indexer` or `dps-server` show up on thec ommand line does not. Data Provisioning Service for what?

For the live indexer, I agree it would be slightly more explicit, but I don't think it outweighs the concern of having binary names that become too long, especially in view of the previous comments. `indexer` is imo clear for past sporks, as it _only_ indexes. The `live` tool, on the other hand, does a lot _more_ than that, so it's not an `indexer`. The `live` word has all the relevant connotations, and referring to it as the `Flow DPS Live` tool makes perfect sense imo.

Reply:
From the user's perspective, what does the Live indexer does that the static one does not, besides connecting to an external IP? 🤔

I feel like yes, from our perspective, we can deduce what Flow DPS Live does based on its name, but from an external perspective I don't think it's so obvious.

As @Maelkum said, I think removing the `dps-` prefix makes more sense than removing `flow-`. Since what our tool does is not done by other ones within the ecosystem, I don't think we'd risk having a conflict in naming between our `flow-XXX` tools and other flow tools, and it would buy us precious space for making things more explicit.

If you still disagree, feel free to mark the conversation as resolved and roll with the current naming, but I strongly feel like it's not explicit enough and might be confusing for users 😄

Reply:
Yes, I do still disagree. Having `flow-server` implies that it is some kind of server for Flow, the main project. It is not, it's the server for the Flow DPS, which is a different project. There are many things that could be a Flow Server. So that would actually make it significantly more confusing.

I do not feel like `flow-dps-live` will be confusing. It's quite clear what `live` means in terms of blockchain networks.